labgrid/remote/client.py

```diff
@@ -1235,7 +1233,10 @@ async def export(self, place, target):
 def start_session(url, realm, extra):
     from autobahn.asyncio.wamp import ApplicationRunner
 
-    loop = asyncio.get_event_loop()
+    loop = asyncio.new_event_loop()
```

Comment:
Doesn't this break use-cases where the caller already has a running event loop?

Reply:
This is not the case in our codebase. But it could be the case in third-party code, though. So maybe add a loop kwarg and use that if it's specified?

Reply:
Some context for the Python upstream change: https://github.com/python/cpython/issues/83710